lib/screens/settings_screen.dart

```diff
@@ -32,7 +32,7 @@ class SettingsScreen extends StatelessWidget {
     _emailAuth.signOut().then((_) => Navigator.pushAndRemoveUntil(
         context,
         MaterialPageRoute<Widget>(
-            builder: (BuildContext context) => RootPage()),
+            builder: (BuildContext context) => RootScreen()),
         (Route<void> route) => false));
```

Comment:
This wasn't changed to the new route generator, was there a reason why?

Reply:
Seems like you've missed this route?